airbyte-integrations/connectors/destination-databricks/src/main/kotlin/io/airbyte/integrations/destination/databricks/jdbc/DatabricksSqlGenerator.kt

```diff
@@ -17,34 +22,29 @@ import io.airbyte.integrations.base.destination.typing_deduping.StreamId
 import io.airbyte.integrations.base.destination.typing_deduping.Struct
 import io.airbyte.integrations.base.destination.typing_deduping.Union
 import io.airbyte.integrations.base.destination.typing_deduping.UnsupportedOneOf
-import io.airbyte.protocol.models.AirbyteRecordMessageMetaChange.*
+import io.airbyte.protocol.models.AirbyteRecordMessageMetaChange.Change
+import io.airbyte.protocol.models.AirbyteRecordMessageMetaChange.Reason
 import io.airbyte.protocol.models.v0.DestinationSyncMode
-import io.github.oshai.kotlinlogging.KotlinLogging
 import java.time.Instant
-import java.util.*
+import java.util.Optional
 
 class DatabricksSqlGenerator(
     private val namingTransformer: NamingConventionTransformer,
     private val unityCatalogName: String,
 ) : SqlGenerator {
 
-    private val log = KotlinLogging.logger {}
     private val cdcDeletedColumn = buildColumnId(CDC_DELETED_COLUMN_NAME)
     private val metaColumnTypeMap =
         mapOf(
             buildColumnId(AB_RAW_ID) to AirbyteProtocolType.STRING,
             buildColumnId(AB_EXTRACTED_AT) to AirbyteProtocolType.TIMESTAMP_WITH_TIMEZONE,
-            buildColumnId(AB_META) to AirbyteProtocolType.STRING
+            buildColumnId(AB_META) to AirbyteProtocolType.STRING,
+            buildColumnId(AB_GENERATION) to AirbyteProtocolType.INTEGER,
         )
 
     companion object {
         const val QUOTE = "`"
         const val CDC_DELETED_COLUMN_NAME = "_ab_cdc_deleted_at"
-        const val AB_RAW_ID = constants.COLUMN_NAME_AB_RAW_ID
-        const val AB_EXTRACTED_AT = constants.COLUMN_NAME_AB_EXTRACTED_AT
-        const val AB_LOADED_AT = constants.COLUMN_NAME_AB_LOADED_AT
-        const val AB_DATA = constants.COLUMN_NAME_DATA
-        const val AB_META = constants.COLUMN_NAME_AB_META
```

Comment:
I could be convinced to undo this, but defining consts to alias other constants feels weird - I replaced those with just import aliases.

(also, fyi `constants` was previously an import alias for JavaBaseConstants)